acceptance/test_run_scripts/555-puppetserver-service/reboot-foss-rhel7.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+SCRIPT_PATH=$(pwd)
+BASENAME_CMD="basename ${SCRIPT_PATH}"
+SCRIPT_BASE_PATH=`eval ${BASENAME_CMD}`
+if [ $SCRIPT_BASE_PATH = "555-puppetserver-service" ]; then
+  cd ../../..
+fi
```

Comment:
What's this trying to do?  I gather change directories to somewhere more appropriate, but I can't figure out the reason why.